src/components/MetaHeadEmbed.tsx

```diff
@@ -66,8 +66,10 @@ export interface MetaEmbedProps {
   /** Webpage description. Should be less than 160 characters. */
   description: string;
 
-  /** Canonical URL of your webpage that will be used as its default app URL. */
-  canonicalUrl?: string;
+  /**
+   * Disable canonical if not desired, defaults to false.
```

Comment:
```suggestion
   * Disable canonical if not desired, defaults to `false`.
```
False is a state so should be surrounded with '`'.